roles/shiftstack/tasks/exec_command_in_pod.yml

```diff
@@ -21,13 +21,13 @@
         PATH: "{{ cifmw_path }}"
         KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
       ansible.builtin.command:
-        cmd: "oc rsh -n {{ namespace }} {{ pod }} bash -c '{{ command }}'"
-      register: command_result
+        cmd: "oc rsh -n {{ namespace }} {{ pod_name }} bash -c '{{ command }}'"
```

Comment:
Command/Shell injection.... This is word a bug in Jira.

Reply:
We decided to go with this as we need to run ansible from the pod (another inventory, hosts and so on) and we didn't want to change ci-framework every time we do some change in our automation repo (shiftstack-qa).
In any case I'm open to discuss alternatives to the command injection.